api-guides/e-receipts.markdown

```diff
@@ -3,142 +3,179 @@ title: eReceipts Recipe
-
-```json
-{
-  "Type": "Ride",
-  "TransactionDateTime": "2014-06-16T16:03:24",
-  "Amount": 88.6,
-  "CurrencyCode": "USD",
-  "MatchingFact": {
-    "Type": "OAuth",
-    "Value": "<insert access token here>"
-  },
-  "Merchant": {
-    "Name": "RideMerchantName",
-    "Location": {
-      "Name": "RideMerchantLocationName",
-      "Address": "RideMerchantAddress",
-      "Address2": "RideMerchantAddress2",
-      "City": "RideMerchantCity",
-      "CountrySubdivisionCode": "WA",
-      "CountryCode": "US",
-      "PostalCode": "RPostalCode",
-      "InternetAddress": "http://ridemerchanturl",
-      "EmailAddress": "ridemerchant@email",
-      "TelephoneNumber": "800-RMERCPH",
-      "FaxNumber": "800-RMERCFX"
-    }
-  },
-  "FormofPaymentCode": "CCARD",
-  "PaymentCard": {
-    "Type": "VI",
-    "MaskedNumber": "4321",
-    "AuthorizationCode": "PaymentAuthCode"
-  },
-  "RideDetail": {
-    "StartDateTime": "2014-06-10T15:08:24.637928-07:00",
-    "EndDateTime": "2014-06-10T15:48:24.637928-07:00",
-    "DriverName": "RideDriverName",
-    "VehicleNumber": "RideVehicleNumber",
-    "DropoffLatitude": 47.4436655,
-    "DropoffLongitude": -122.2982266,
-    "LineItems": [
-      {
-        "SequenceNumber": 1,
-        "Reference": "RideRefId01",
-        "Description": "Ride Item 01 Description",
-        "Description2": "Ride Item 01 Description2",
-        "Quantity": 1,
-        "Amount": 73.65
-      },
-      {
-        "SequenceNumber": 2,
-        "Reference": "RideRefId02",
-        "Description": "Ride Item 02 Description",
-        "Description2": "Ride Item 02 Description2",
-        "Quantity": 1,
-        "Amount": 4.95
-      },
-      {
-        "SequenceNumber": 3,
-        "Reference": "RideRefId03",
-        "Description": "Ride Item 03 Description",
-        "Description2": "Ride Item 03 Description2",
-        "Quantity": 1,
-        "Amount": 10
-      }
-    ]
-  },
-  "CustomFields": [
-    {
-      "Name": "RideCustomField01Name",
-      "Value": "RideCustomField02Value"
-    },
-    {
-      "Name": "RideCustomField02Name",
-      "Value": "RideCustomField02Value"
-    },
-    {
-      "Name": "RideCustomField03Name",
-      "Value": "RideCustomField03Value"
-    }
-  ],
-  "ImageBase64": "<insert Base64 representation of image>"
-}
-```
-
-Test your API in the [API Explorer](/api-explorer/v3-0/Receipts.html).  
-
-
-#### If the user connection is initiated from the Partner site, it should follow Web Flow:
-
-![Web Flow](images/web_flow.JPG)
-
-#### If the connection is initated from the Concur App Center, the user experience should follow App Center Flow:
-
-![App Center Flow](images/Normal-flow.jpg)
+### Getting Started
+
+**New App Center Partners:**  Your Company must have entered into a commercial agreement with Concur.  If you have not yet entered into said agreement with Concur, click [here](https://developer.concur.com/why-concur.html) to contact our business development team
```

Comment:
Since we have a separate preview site (https://preview.developer.concur.com), any links to Dev Center should omit the host. Ex. `click [here](/why-concur.html)`